NuRadioMC/utilities/attenuation.py

```diff
@@ -259,9 +245,63 @@ def get_attenuation_length(z, frequency, model):
     else:
         att_length_f[att_length_f < min_length] = min_length
         att_length_f[z > 0] = np.inf
+
     return att_length_f
 
 
+try:
```

Comment:
Just to understand - these are just jitted versions of the `scipy.interpolate` classes defined at the top of the file? I would have complained about the depth sign convention but if this is just to ensure these functions have the same output as `__call__` on the already-defined classes, I think it's fine to overwrite them. Just feels a bit odd (I guess one could consider making all of these private, too).

Reply:
...and I guess you've checked that using a jitted version of `np.interpolate` is faster than the scipy class (I don't need exact numbers, just would be good that there's a point to doing this)?

Reply:
depth being positive, I'm not a fan because we generally work with `z<0`. But seeing as this is (or should be) an internal function I can live with it.